### PR TITLE
clair: init at 2.0.7

### PR DESCRIPTION
**Pull request by marsam**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/admin/clair/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  goPackagePath = "github.com/coreos/clair";
+
+  src = fetchFromGitHub {
+    owner = "coreos";
+    repo = "clair";
+    rev = "v${version}";
+    sha256 = "0n4pxdw71hd1rxzgf422fvycpjkrxxnvcidys0hpjy7gs88zjz5x";
+  };
+
+  buildInputs = [ makeWrapper ];
```
Comment (Ma27) [inline pkgs/tools/admin/clair/default.nix]: The `makeWrapper` dependecy should be in `nativeBuildInputs`. The purpose is explained in the [NixOS manual](https://nixos.org/nixpkgs/manual/#ssec-stdenv-dependencies).

**Comment by Ma27** [comment]
built and tested locally. Except for my comment this seems fine :+1:

File: pkgs/tools/admin/clair/default.nix (hunk 1 above)
Comment (marsam) [reply pkgs/tools/admin/clair/default.nix]: Fixed, thanks!

**Comment by Ma27** [comment]
@marsam thanks!

**Pull request merged**